packages/system/velero/charts/velero/templates/role.yaml

```
@@ -0,0 +1,21 @@
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/component: server
+    app.kubernetes.io/name: {{ include "velero.name" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    helm.sh/chart: {{ include "velero.chart" . }}
+rules:
+- apiGroups:
+    - "*"
+  resources:
+    - "*"
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)